### DISCUSSION TITLE
How to prevent users logged in with decap-cms from seeing the source code of the website?

### DISCUSSION THREAD

**Discussion by ljzhsq2** [decaporg/decap-cms#7213] [Q&A]
Hello everyone, I like decap-cms, but there is a problem that I don't know how to solve?

My decap-cms has many users who can log in to jointly maintain the hugo website.

Since I am logging in using github, I need to grant everyone's github user access to the warehouse. They can easily see the source code of all hugo websites in github after logging in, but I don't want them to see the hugo website source code. How to solve it?

If I use **[Git Gateway without Netlify]** to log in, it can indeed be solved, but this method is very troublesome. Is there a simpler way?

**Comment by kylesloper** [comment]
Unfortunately this is not a feature request. You need to do this via the Github backend or create your own API as the backend.
